fheroes2.appdata.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>fheroes2.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>fheroes2</name>
+  <summary>Free Heroes of Might and Magic II (fheroes2) is a recreation of HoMM2 game engine.</summary>
+  <description>
+    <p>Free Heroes of Might and Magic II (fheroes2) is a recreation of HoMM2 game engine.</p>
```

Comment:
Hi! Could you please change these lines as well to match the actual name of the project? Thank you!
```suggestion
  <summary>fheroes2 is a recreation of HoMM2 game engine.</summary>
  <description>
    <p>fheroes2 is a recreation of HoMM2 game engine.</p>
```

Reply:
Of course. 😉